docs/src/main/mdoc/quick-example.md

```diff
@@ -37,12 +37,17 @@ object Main extends IOApp {
           processRecord(committable.record)
             .map { case (key, value) =>
               val record = ProducerRecord("topic", key, value)
-              ProducerRecords.one(record, committable.offset)
+              committable.offset -> ProducerRecords.one(record)
             }
-        }
-        .through(KafkaProducer.pipe(producerSettings))
-        .map(_.passthrough)
-        .through(commitBatchWithin(500, 15.seconds))
+        }.through { offsetsAndProducerRecords =>
+          KafkaProducer.stream(producerSettings).flatMap { producer =>
+            offsetsAndProducerRecords.evalMap { 
+              case (offset, producerRecord) => 
+                producer.produce(producerRecord)
+                  .map(_.as(offset))
+            }.parEvalMap(Int.MaxValue)(identity)
```

Comment:
This is a bit awkward - I wonder if we could convince the fs2-kafka maintainers to add `parFlattenUnbounded` 🤔

Reply:
If not, maybe at least `parEvalMapUnbounded`?

Reply:
I opened a PR for `parEvalMapUnbounded` - https://github.com/typelevel/fs2/pull/2849.